src/node.cc

```diff
@@ -1923,6 +1923,63 @@ static void Uptime(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetResourceUsage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err)
+    return env->ThrowUVException(err, "uv_getrusage");
+
+  Local<Object> resource_usage = Object::New(env->isolate());
```

Comment:
Virtual shrug.  I don't think it really matters here.

`fs.stat()` gets called thousands of times at start-up so it makes sense to optimize it to death.  This getrusage() wrapper is not nearly as performance sensitive.